packages/react/src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -122,6 +127,7 @@ export const AnchoredOverlay: React.FC<React.PropsWithChildren<AnchoredOverlayPr
   anchorOffset,
   className,
   pinPosition,
+  variant = {regular: 'anchored', narrow: 'anchored'},
```

Comment:
nit: should we support wide as well to follow suit with other components? or is there precedence in PRC for doing only some viewports? 🤔

Reply:
You're right, we should include it as well. (I think the default works even if you don't because it's all anchored)

```suggestion
  variant = {regular: 'anchored', narrow: 'anchored', wide: 'anchored'},
```